### PR TITLE
Kafka 3.2 bump

### PR DESCRIPTION
**Pull request by arkadius**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -14,18 +14,9 @@ import scala.util.Try
 import scala.xml.Elem
 import scala.xml.transform.{RewriteRule, RuleTransformer}
 
-// Warning: Flink doesn't work correctly with 2.12.11
-// Warning: 2.12.13 + crossVersion break sbt-scoverage: https://github.com/scoverage/sbt-scoverage/issues/319
-val scala212 = "2.12.10"
+val scala212 = "2.12.16"
```
Comment (mproch) [inline build.sbt:17]: I think scala bump is still not possible: https://lists.apache.org/thread/hwksnsqyg7n3djymo7m1s7loymxxbc3t

Comment (arkadius) [reply build.sbt]: still? what's a pity. ok, I've removed scala upgrade from this PR

File: security/src/test/scala/pl/touk/nussknacker/ui/security/oidc/OidcServiceSpec.scala [hunk 2]
```diff
@@ -64,7 +64,6 @@ class OidcServiceSpec extends FunSuite with Matchers with EitherValuesDetailedMe
 
     inside(validator.introspect[String](validToken)) {
       case Invalid(err: OAuth2JwtDecodeClaimsJsonError) =>
-        err.msg should not include "Foo Bar"
```
Comment (mproch) [inline security/src/test/scala/pl/touk/nussknacker/ui/security/oidc/OidcServiceSpec.scala:67]: it doesn't work? or is it irrelevant?

**Review by mproch — COMMENTED** [review]
looks good, maybe MigrationGuide entry will also be helpful... this kafka bump is quite large ;)

**Review by mproch — APPROVED** [review]
cool :+1:

**Pull request merged**